src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    private static final double defaultRadius = 10.0;
+    private static final String defaultColor = "WHITE";
```

Comment:
```suggestion
    private static final double DEFAULT_RADIUS = 10.0;
    private static final String DEFAULT_COLOR = "WHITE";
```
Don't forget about what we should call constants